tests/unit/completion/test_completionwidget.py

```diff
@@ -0,0 +1,152 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2016 Ryan Roden-Corrent (rcorre) <ryan@rcorre.net>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the CompletionView Object."""
+
+import unittest.mock
+
+import pytest
+from PyQt5.QtGui import QStandardItem, QColor
+
+from qutebrowser.completion import completionwidget
+from qutebrowser.completion.models import base, sortfilter
+from qutebrowser.utils import objreg
+
+
+@pytest.yield_fixture
+def completionview(qtbot, status_command_stub, config_stub, win_registry,
+                   monkeypatch):
```

Comment:
you'll also need to request the `mocker` fixture instead of `monkeypatch` here :wink: